### PR TITLE
Added Keyboard Shortcuts For Opacity

### PR DESCRIPTION
**Pull request by The-Code-Monkey**
cmd || ctrl & +: increase opacity by 10 until 100%.
cmd || crtl & -: decrease opacity by 10 until 20%

Works on mac, unknown if works on mobile should work though if someone has a windows pc to test on

### CODE REVIEW

File: public/menu.js [hunk 1]
```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
...
+        {
+          label: 'Lower Opacity',
+          accelerator: 'CmdOrCtrl+-',
+          click() {
+            mainWindow.setOpacity(minus(mainWindow));
+          }
+        },
+        {
+          label: 'Increase Opacity',
+          accelerator: 'CmdOrCtrl+=',
```
Comment (mubaidr) [inline public/menu.js]: Should not this be `CmdOrCtrl++`?

Comment (The-Code-Monkey) [reply public/menu.js]: Nope as the key is known to js as = not as +. + is a concatination key

Comment (The-Code-Monkey) [reply public/menu.js]: so its not able to be used. if you wanted it to be plus you would have to do CmdOrCtrl+Shift+=

Comment (The-Code-Monkey) [reply public/menu.js]: I thought it would have been + at first but it does nothing.

**Comment by The-Code-Monkey** [comment]
@kamranahmedse is it possible to merge this or is there something you would like me to change?

File: public/menu.js [hunk 2]
```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
     {
       label: 'Edit',
       submenu: [
+        {
+          label: 'Lower Opacity',
+          accelerator: 'CmdOrCtrl+-',
+          click() {
+            mainWindow.setOpacity(minus(mainWindow));
```
Comment (kamranahmedse) [inline public/menu.js]: Change this to `getUpdatedOpacity(mainWindow.getOpacity(), -0.1)`

File: public/menu.js [hunk 3]
```diff
@@ -1,12 +1,30 @@
-const { app, Menu, shell } = require('electron');
+const { app, Menu, shell, ipcRenderer } = require('electron');
```
Comment (kamranahmedse) [inline public/menu.js]: Remove this `ipcRenderer` import

File: public/menu.js [hunk 4]
```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
...
+          accelerator: 'CmdOrCtrl+-',
+          click() {
+            mainWindow.setOpacity(minus(mainWindow));
+          }
+        },
+        {
+          label: 'Increase Opacity',
+          accelerator: 'CmdOrCtrl+=',
+          click() {
+            mainWindow.setOpacity(plus(mainWindow));
```
Comment (kamranahmedse) [inline public/menu.js]: Change this to `getUpdatedOpacity(mainWindow.getOpacity(), 0.1)`

File: public/menu.js (hunk 1 above)
Comment (kamranahmedse) [reply public/menu.js]: Change this shortcut to `CmdOrCtrl+Up`

File: public/menu.js [hunk 5]
```diff
@@ -23,6 +41,20 @@ function setMainMenu() {
     {
       label: 'Edit',
       submenu: [
+        {
+          label: 'Lower Opacity',
+          accelerator: 'CmdOrCtrl+-',
```
Comment (kamranahmedse) [inline public/menu.js]: Change this shortcut to `CmdOrCtrl+Down`

**Comment by kamranahmedse** [comment]
Thanks @The-Code-Monkey, looks good. 
Just look at the comments I left and we will be good to go.

File: public/menu.js (hunk 5 above)
Comment (eeshaan) [reply public/menu.js]: I agree. `CmdOrCtrl+=` and `CmdOrCtrl+-` can be interpreted as zoom shortcuts.

File: public/menu.js [hunk 6]
```diff
@@ -1,49 +1,91 @@
...
+    {
+      label: "Tools",
       submenu: [
-        { role: 'undo' },
-        { role: 'redo' },
-        { type: 'separator' }, //just adds a line visually
-        { role: 'cut' },
-        { role: 'copy' },
-        { role: 'paste' },
-        { role: 'selectall' }
```
Comment (kamranahmedse) [inline public/menu.js]: Please revert the quotes to be single

**Comment by kamranahmedse** [comment]
Thank you

**Comment by The-Code-Monkey** [comment]
No worries happy to help.

**Comment by The-Code-Monkey** [comment]
When will you rebuild the project and push release 0.1.1

**Pull request merged**